backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/MigrateMultipleVmsCommand.java

```diff
@@ -102,11 +102,12 @@ protected boolean validate() {
             return false;
         }
 
+        List<String> outputMessages = new ArrayList<>();
         Map<Guid, List<VDS>> possibleVmHosts = schedulingManager.prepareCall(getCluster())
                 .hostBlackList(getHostBlackList())
                 .hostWhiteList(getHostWhiteList())
                 .ignoreHardVmToVmAffinity(getParameters().isCanIgnoreHardVmAffinity())
-                // TODO - Use error messages from scheduling
+                .outputMessages(outputMessages)
```

Comment:
if the validation messages are not used when we return true here (in the validation method), how about passing getReturnValue().getValidationMessages() as we do in MigrateVm and RunVm?